src/frontend/views/modules/header/header.pug

```diff
@@ -6,7 +6,7 @@ div
       ul.list-inline.all-centered
         li.logo
           a(href='', ui-sref='home')
-            img.openpaas-logo(ng-src="{{'/api/themes/' + domainId + '/logo'}}", fallback-src='/images/white-logo.svg', src="/images/white-logo.svg", alt="{{ 'OpenPaas Logo' | translate }}")
+            img.openpaas-logo(ng-src="{{apiURL + '/api/themes/' + domainId + '/logo'}}", fallback-src='/images/white-logo.svg', src="/images/white-logo.svg", alt="{{ 'OpenPaas Logo' | translate }}")
```

Comment:
we also need to do it for the `favicon` here https://github.com/linagora/esn-frontend-common-libs/blob/43972a3af0775899526b030cb32e94b011f23300/src/frontend/views/modules/sidebar/contextual-sidebar.pug#L10